core/runtime/src/main/java/io/quarkus/runtime/ConfigConfig.java

```diff
@@ -74,14 +74,6 @@ public interface ConfigConfig {
     @WithDefault("warn")
     BuildTimeMismatchAtRuntime buildTimeMismatchAtRuntime();
 
-    /**
-     * A property that allows accessing a generated UUID.
-     * It generates that UUID at startup time. So it changes between two starts including in dev mode.
-     * <br>
-     * Access this generated UUID using expressions: `${quarkus.uuid}`.
-     */
-    Optional<String> uuid();
-
```

Comment:
Does it actually change something to remove this? Asking because it's the only way to document it in the config.

Reply:
If a property is declared in a mapping (or the old root), it is always eagerly populated. It wouldn't matter if `quarkus.uuid` is lazy. This is always trigger the load.

Reply:
OK. I suppose we need to make sure it's properly documented elsewhere then.

Reply:
I think we need to do it manually. We don't have a way to do it in the processor. We can ignore properties for documentation, but we can't ignore properties for mapping.